docs/root/configuration/http_conn_man/stats.rst

```diff
@@ -53,8 +53,9 @@ statistics:
    downstream_rq_4xx, Counter, Total 4xx responses
    downstream_rq_5xx, Counter, Total 5xx responses
    downstream_rq_ws_on_non_ws_route, Counter, Total WebSocket upgrade requests rejected by non WebSocket routes
-   downstream_rq_time, Histogram, Request time milliseconds
+   downstream_rq_time, Histogram, Total time for request and response (milliseconds)
    downstream_rq_idle_timeout, Counter, Total requests closed due to idle timeout
+   downstream_rq_timeout, Counter, Total requests closed due to a timeout on the request path
```

Comment:
I think this might not match the stat name that you picked in code. Can you check?